zwave_js_server/model/node/data_model.py

```diff
@@ -14,6 +14,13 @@
     from typing import TypedDict
 
 
+class FoundNodeDataType(TypedDict, total=False):
+    """Represent a found node data dict type."""
+
+    nodeId: int
+    deviceClass: DeviceClassDataType
```

Comment:
What about supported CCs and controlled CCs?

https://zwave-js.github.io/node-zwave-js/#/getting-started/migrating-to-v10?id=changed-argument-type-of-quotnode-foundquot-event

Reply:
We don't currently include these in the server when building the node found event data.

Reply:
added a todo for that https://github.com/zwave-js/zwave-js-server/issues/722